checkmarx/util.py

```diff
@@ -473,3 +485,37 @@ def get_checkmarx_cfg(checkmarx_cfg_name: str) -> cmmmodel.CheckmarxConfig:
 @functools.lru_cache()
 def create_checkmarx_client(checkmarx_cfg: cmmmodel.CheckmarxConfig):
     return checkmarx.client.CheckmarxClient(checkmarx_cfg)
+
+
+def upload_results_to_deliverydb(
```

Comment:
naming-nit: while it is true that the results end up in delivery-db, we actually upload them to delivery-service.